src/api/app/assets/javascripts/webui2/monitor.js

```diff
@@ -0,0 +1,10 @@
+function monitor_ready() { // jshint ignore:line
```

Comment:
How do you know you need this function?

Reply:
Taking into account that we don't plan to change many things, I guess we are going to keep most of this code, maybe with some slight differences.

Reply:
Let's only take over what we actually are going to use please. We have to maintain this code.

Reply:
:+1: done!